client/signup/steps/site-type/form.jsx

```diff
@@ -38,8 +41,10 @@ class SiteTypeForm extends Component {
 	};
 
 	render() {
+		const { showDescriptions, translate } = this.props;
+
 		return (
-			<Fragment>
+			<>
```

Comment:
Nice! I was not aware of the short syntax.

Reply:
We have a mixture of them in the codebase at this point but short syntax seems to be taking over. :-)